app/src/main/java/org/asteroidos/sync/PermissionsActivity.java

```diff
@@ -34,7 +35,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         PackageManager pm = getPackageManager();
         boolean hasBLE = pm.hasSystemFeature(PackageManager.FEATURE_BLUETOOTH_LE);
 
-        if (hasBLE) {
+        ActivityManager am = (ActivityManager) getApplicationContext().getSystemService(ACTIVITY_SERVICE);
+        System.out.println("is low ram: " + am.isLowRamDevice());
+        boolean isLowRamDevice = am.isLowRamDevice();
+
+        if (hasBLE && !isLowRamDevice) {
```

Comment:
What about
if (hasBLE) {
    if (isLowRamDevice) {
        Okay
    } else {
        We don't support android go
    }
} else {
    Ble not supported
}

This would avoid having to check variables multiple times and would simplify your conditions.